exts/Makefile.inc

```diff
@@ -0,0 +1,10 @@
+
+LIBS     := -L$(LIBEXTSDIR)/dbcsr -ldbcsr $(LIBS)
+EXTSDEPS  = $(LIBEXTSDIR)/dbcsr/libdbcsr.a
```

Comment:
@pseewald After the discussion with @oschuett, likely here we can relax this dependency by using the dbcsr_api.mod ($(OBJEXTSDIR)/dbcsr_api.mod). Could you try that?

Reply:
Yes but I think we need two things:
- executables should depend on libdbcsr.a to make sure that they are linked against the changed library
- objects should depend on dbcsr_api.mod

Do you agree?

Reply:
I think it is too much work... Assuming that you change .mod, then I would argue that .a gets recompiled too. In any case, CP2K will get recompiled, so you get it to link again. Therefore, checking for a change in dbcsr_api.mod is enough.

Reply:
As pointed by Patrick, we need two dependencies (compilation by dbcsr_api*.mod and linking by libdbcsr.a). Let's postpone it after the merging.

Reply:
We have several DBCSR *.mod files used in cp2k, not only dbcsr_api:
- dbcsr_vector
- dbcsr_tensor_api
- dbcsr_acc_device

I think we should not hardcode dependencies on mod files then.

Reply:
This has been taken care of with db2c1ccfdca9c2a83d97e946e05d89b05058bf5a
@alazzaro maybe you want to take a quick look and review my changes. I removed copying of mod files from $(OBJEXTSDIR) to $(OBJDIR) since this always updates the timestamp.

Reply:
Thanks Patrick, it makes sense. There is something wrong with the CI (as usual)...

Reply:
The CI is optimized for quick results in the common case. This PR is really the opposite of common. So, please have some mercy ... 

I guess any change to the Makefile should trigger a `make distclean`. I'll look into it.

Reply:
@pseewald I assume that you want your commit 2398b7141b4a96587dcee72ab3e5adcf396f1cb1 into the master release of DBCSR... That's means we have to make a new RC (@dev-zero )

Reply:
Ok, checks are passing again.

Reply:
> @pseewald I assume that you want your commit 2398b7141b4a96587dcee72ab3e5adcf396f1cb1 into the master release of DBCSR... That's means we have to make a new RC (@dev-zero )

Done